source/user-manual/api/getting-started.rst

```diff
@@ -898,6 +898,33 @@ Adding an agent is now easier than ever. Simply send a request with the agent na
       "error": 0
     }
 
+Ingest security events
+^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
Should we add a `.. versionadded:: 4.5.0` tag or is it unnecessary?

Reply:
That is relevant, I will add it.

Reply:
@nico-stefani , `4.5` or `4.6`? Due to latest version changes, there's a mismatch between branch and `versionadded` tag here.

Reply:
@javimed, yes, you are right. Here is the fix 5f24621eb71820637fcab02a1b0bc1477eaad87e